docs/MinimumNode.md

```diff
@@ -0,0 +1,151 @@
+# Minimum Node Service
```

Comment:
Please name this file MinimumNodeService.md as this is the name of the class.

Reply:
Done